run-integration-tests.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+
+#
+# Copyright 2019 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import sys
+import fnmatch
+import subprocess
+from os import path
+import random
+import string
+import tempfile
+
+
+# TODO Scala integration tests
+def run_scala_integration_tests(root_dir, package, repo):
+    pass
+
+
+def run_python_integration_tests(root_dir, package, repo):
+    print("##### Running Python tests #####")
+    test_dir = path.join(root_dir, path.join("examples", "python"))
+    test_files = [os.path.join(test_dir, f) for f in os.listdir(test_dir)
+                  if os.path.isfile(os.path.join(test_dir, f)) and
+                  f.endswith(".py") and not f.startswith("_")]
+    python_root_dir = path.join(root_dir, "python")
+    extra_class_path = path.join(python_root_dir, path.join("delta", "testing"))
```

Comment:
what is this for?

Reply:
its to make sure that INFO logs arent being printed, we did the same for our test suite as well

Reply:
aah okay. Maybe add a comment saying that. this is surprising otherwise.